### DISCUSSION TITLE
Pokéball validity for Colo/XD

### DISCUSSION THREAD

**Discussion by Lorenzooone** [kwsch/PKHeX#3848] [Ideas]
Colosseum (and Pokémon XD???) are special because they can't trade until you have cleared the game. Meaning that certain Pokémon caught in specific locations cannot be in certain Pokéballs, but only in a subset. For example, with the exception of JP Colosseum and the JP Bonus disc, it means that you cannot catch an Entei on Mt. Battle with a Master Ball. You can do that in The Deep Colosseum, but not in Mt. Battle.
PkHex currently reports the case above as legal, though.
I understand it's not an easy thing to do, and it may not be worth it, but I'd like to point attention to Pokéball legality in Colo/XD...

**Comment by kwsch** [comment]
Related:
https://github.com/kwsch/PKHeX/issues/3795
